internal/storage/runtime.go

```diff
@@ -346,12 +348,12 @@ func (r *runtimeService) createContainerOrPodSandbox(systemContext *types.System
 	}, nil
 }
 
-func (r *runtimeService) CreatePodSandbox(systemContext *types.SystemContext, podName, podID, imageName, imageAuthFile, imageID, containerName, metadataName, uid, namespace string, attempt uint32, idMappings *idtools.IDMappings, labelOptions []string, privileged bool) (ContainerInfo, error) {
-	return r.createContainerOrPodSandbox(systemContext, podName, podID, imageName, imageAuthFile, imageID, containerName, podID, metadataName, uid, namespace, attempt, idMappings, labelOptions, true, privileged)
+func (r *runtimeService) CreatePodSandbox(systemContext *types.SystemContext, podName, podID, imageName, imageAuthFile, imageID, containerName, metadataName, uid, namespace string, attempt uint32, idMappingsOptions *storage.IDMappingOptions, labelOptions []string, privileged bool) (ContainerInfo, error) {
```

Comment:
This is not related to this PR, but the number argument `CreatePodSandbox` takes unusually high. Can we not have some of these embedded inside `*runtimeService` or maybe we can have `CreateSandoxInfo` struct to hold those values and pass it here as an argument. 

@haircommander WDYT?

Reply:
@wgahnagl is in the process of refactoring a lot of the sandbox/container creation code into new interfaces. I think it is a good idea to refactor the arguments here, but I think it should be `pkg/sandbox/sandbox`, not another intermediate struct